vcfa/resource_vcfa_supervisor_namespace_test.go

```diff
@@ -0,0 +1,193 @@
+				ImportStateVerifyIgnore: []string{"name_prefix"},
+				ImportStateIdFunc: func(s *terraform.State) (string, error) {
+					return params["ProjectName"].(string) + ImportSeparator + cachedNamespaceName.fieldValue, nil
+				},
+			},
+			{
+
+				Config: configText4,
+				Check: resource.ComposeTestCheckFunc(
+					cachedNamespaceName.testCheckCachedResourceFieldValuePattern("data.vcfa_kube_config.test-namespace", "id", fmt.Sprintf("%s:%%s:%s", testConfig.Org.Name, params["ProjectName"].(string))),
+					cachedNamespaceName.testCheckCachedResourceFieldValuePattern("data.vcfa_kube_config.test-namespace", "context_name", fmt.Sprintf("%s:%%s:%s", testConfig.Org.Name, params["ProjectName"].(string))),
+					resource.TestCheckResourceAttr("data.vcfa_kube_config.test-namespace", "insecure_skip_tls_verify", fmt.Sprintf("%t", testConfig.Provider.AllowInsecure)),
+					resource.TestCheckResourceAttr("data.vcfa_kube_config.test-namespace", "user", fmt.Sprintf("%s:%s@%s", testConfig.Org.Name, testConfig.Org.User, ref.Host)),
+					resource.TestCheckResourceAttrSet("data.vcfa_kube_config.test-namespace", "token"),
+					resource.TestCheckResourceAttrSet("data.vcfa_kube_config.test-namespace", "kube_config_raw"),
+				),
+			},
+		},
+	})
+
+	postTestChecks(t)
+}
+
+const testAccVcfaSupervisorNamespaceStep1 = `
+resource "vcfa_supervisor_namespace" "test" {
+  name_prefix  = "terraform-test"
+  project_name = "{{.ProjectName}}"
+  class_name   = "small"
+  description  = "Supervisor Namespace created by Terraform"
+  region_name  = "{{.RegionName}}"
+  vpc_name     = "{{.VpcName}}"
+
+  storage_classes_initial_class_config_overrides {
+    limit_mib = 200
+    name      = "{{.StorageClassName}}"
+  }
+
+  zones_initial_class_config_overrides {
+    cpu_limit_mhz          = 100
+    cpu_reservation_mhz    = 1
+    memory_limit_mib       = 200
+    memory_reservation_mib = 2
+    name                   = "{{.SupervisorZoneName}}"
+  }
+}
+`
+
+const testAccVcfaSupervisorNamespaceStep2DS = testAccVcfaSupervisorNamespaceStep1 + `
+data "vcfa_supervisor_namespace" "test" {
+  name         = vcfa_supervisor_namespace.test.name
+  project_name = vcfa_supervisor_namespace.test.project_name
+}
+`
+
+const testAccVcfaSupervisorNamespaceStep4KubeConfig = testAccVcfaSupervisorNamespaceStep1 + `
+data "vcfa_supervisor_namespace" "test" {
+  name         = vcfa_supervisor_namespace.test.name
+  project_name = vcfa_supervisor_namespace.test.project_name
+}
+
+data "vcfa_kube_config" "test-namespace" {
+  project_name              = vcfa_supervisor_namespace.test.project_name
+  supervisor_namespace_name = vcfa_supervisor_namespace.test.name
+}
+`
+
+func setupProject(t *testing.T, projectName string) func() {
+	// setup project
+	tmClient := createTemporaryVCFAConnection(false)
+
+	projectCfg := &ccitypes.Project{
+		TypeMeta: v1.TypeMeta{
+			Kind:       ccitypes.ProjectKind,
+			APIVersion: ccitypes.ProjectCciAPI + "/" + ccitypes.ApiVersion,
```

Comment:
`ProjectVersion` instead of `ApiVersion`

Reply:
Fixed.